docs/docs/noir/concepts/generics.md

```diff
@@ -161,3 +154,20 @@ fn example() {
     assert(10 as u32 == foo.generic_method::<Field>());
 }
 ```
+
+### Numeric Generics
+
+Numeric generics are specified using the `let NAME: TYPE` syntax, and are
+required when a generic `N` is used for the size of an array (`[T; N]`) or string (`str<N>`):
+
+```rust
+fn double<let N: u32>() -> u32 {
+    N * 2
```

Comment:
Can you introduce this earlier on this page? The `BigInt<N>` example needs to be updated to use explicit numeric generics as well.